src/traversals/bipartiteness.jl

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
Sure. I was trying to make things nouns as opposed to adjectives, though. I don't much care :)